Classes/game.py

```diff
@@ -0,0 +1,34 @@
+from card import Card
+from random import shuffle
+
+class Game:
```

Comment:
I'm basically thinking this class can represent the game state as a whole.  It'll have a board class, market, etc..
This is what every persons program will be able to see, and then the players turn will update this state